cl/favorites/templates/top_prayers.html

```diff
@@ -63,7 +63,7 @@ <h3 class="text-center v-offset-below-3"><b>{{ waiting_stats.prayer_count|intcom
                 </a>
               </td>
               <td>
-                <a href="{% url "view_recap_document" docket.pk prayer.docket_entry.entry_number docket.slug %}">
+                <a href="{{ prayer.get_absolute_url }}" rel="nofollow">
```

Comment:
The document link fix is working correctly. However, the table becomes cluttered when multiple attachments from the same entry are listed consecutively, as shown in this screenshot:

<img width="1167" alt="Screenshot 2025-03-28 at 10 16 03 PM" src="https://github.com/user-attachments/assets/47451ef1-dc1d-414b-8c36-a8675c8787db" />

To improve readability, I suggest adding the attachment number to each record when available. This would help distinguish individual attachments: 

<img width="1167" alt="Screenshot 2025-03-28 at 10 20 26 PM" src="https://github.com/user-attachments/assets/d2bd005c-6436-4c38-931d-a06ee8a025e7" />

Let me know what you think

Reply:
Here's the code for the template: 

```
                <a href="{{ prayer.get_absolute_url }}" rel="nofollow">
                  {% if prayer.attachment_number %}
                    {{ prayer.document_number }}-{{ prayer.attachment_number }}
                  {% else%}
                    {{ prayer.document_number }}
                  {% endif %}
                </a>
```